cli/src/args.ts

```diff
@@ -19,21 +18,21 @@ type BenchmarkUnsafeArguments = {
   help?: boolean
 }
 
-const argExample = 'e.g. npm run benchmark -- -u 2 -e 3, runs 2 users for 3 epochs'
+const argExample = 'e.g. npm start -- -u 2 -e 3 # runs 2 users for 3 epochs'
 
 const unsafeArgs = parse<BenchmarkUnsafeArguments>(
   {
-    task: { type: String, alias: 't', description: 'Task', defaultValue: 'simple_face' },
+    task: { type: String, alias: 't', description: 'Task: titanic, simple_face or cifar10', defaultValue: 'simple_face' },
     numberOfUsers: { type: Number, alias: 'u', description: 'Number of users', defaultValue: 1 },
     epochs: { type: Number, alias: 'e', description: 'Number of epochs', defaultValue: 10 },
     roundDuration: { type: Number, alias: 'r', description: 'Round duration', defaultValue: 10 },
-    batchSize: { type: Number, alias: 'b', description: 'Round duration', defaultValue: 10 },
+    batchSize: { type: Number, alias: 'b', description: 'Training batch size', defaultValue: 10 },
     save: { type: Boolean, alias: 's', description: 'Save logs of benchmark', defaultValue: false },
     help: { type: Boolean, optional: true, alias: 'h', description: 'Prints this usage guide' }
   },
   {
     helpArg: 'help',
-    headerContentSections: [{ header: 'Disco benchmark', content: 'npm run benchmark -- [Options]\n' + argExample }]
+    headerContentSections: [{ header: 'DISCO CLI', content: 'npm run start -- [Options]\n' + argExample }]
```

Comment:
both are valid but consistency is good

```suggestion
    headerContentSections: [{ header: 'DISCO CLI', content: 'npm start -- [Options]\n' + argExample }]
```